### DISCUSSION TITLE
Next app at app.example.com, but I want it to be at example.com/login, example.com/search etc (but Webflow site is on example.com)

### DISCUSSION THREAD

**Discussion by founderpad** [vercel/vercel#6993] [Help]
Is this possible?

I have a Webflow site (landing page, legals, privacy policy etc) at example.com and the Nextjs app at app.example.com (Vercel).

I really need to serve the app at example.com as well, but the Webflow site is there.

I tried a rewrite but it just gave me a blank page, and I have a feeling it's probably more involved than just a URL rewrite anyway but I'm a complete noob with this stuff.

Is this possible to do?

**Comment by leerob** [comment]
Yes! Rewrites should be the solution. For example, you could rewrite `/legal` on your Vercel deployment to your external Webflow URL (e.g. `https://mywebflowsite.com/legal`). The visible URL won't change - it proxies to the Webflow deployment behind the scenes.
